chia/util/timing.py

```diff
@@ -23,9 +23,10 @@
 
 if os.environ.get("GITHUB_ACTIONS") == "true":
     # https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables
```

Comment:
I don't think you meant to change the entire logic of this , you are returning the "local" setting when the env var is set and not setting anything otherwise?